contents/teams/error-tracking/objectives.mdx

```diff
@@ -1,56 +1,24 @@
-### Q4 2025 Objectives
+### Q1 2026 Objectives
 
-#### Go all in on AI (Driver: <TeamMember name="David Newell" />)
-* Implement issue autofixing
-  * In-app flow where an issue can be assigned to Max
-  * Build terminal based experience within the CLI
-    * Add issue selection to MCP (e.g. by file, most impactful)
-    * Stretch: Add test writing agent to prevent regressions
-* Add exception capture auto instrumentation to the wizard
-* Integrations to capture exceptions for popular LLM / agent tools (e.g. Langchain)
+#### **Alerting** - Spikes & Digest (Driver: <TeamMember name="Aleksander Błaszkiewicz" />)
+* Define how we calculate a spike and what are the controls
+* Add 'Issue Spiked' trigger in alerting
+* Add ability to snooze issues from alerting
+* Create weekly digest (warnings, resolved, recommendations, priorities...)
+* Stretch: correlate with event traffic to understand the context of a “spike”
 
-#### Ship mobile error tracking (Driver: <TeamMember name="Manoel Aranda Neto" />)
-* Complete 'straightforward' section of the [megaissue](https://github.com/PostHog/posthog/issues/38845)
-* Support suppression rules
-* Add suppression of minified exceptions before cymbal processing
-* Build dashboard template including crash free sessions / users metric
+#### **Grouping** - Fingerprint & Sourcemaps (Driver: <TeamMember name="Hugues Pouillot" />)
+* Iterate on fingerprint algorithm using labelling tool
+* Stretch: Add bundler plugins to make sourcemap uploads easier
+* Stretch: Explore embedding based grouping
 
-#### Research deep dive on customers (Driver: <TeamMember name="Cory Slater" />)
-* Interview existing, potential, and recently churned customers to understand:
-  * What makes people switch from competitors and what pulls them back?
-  * What value would convince customers to remove billing limits?
-  * How do AI-first companies handle errors differently?
-  * What context would let customers skip steps in their workflow? 
-  * What drives month to month swings in spend vs contraction?
-* Every engineer to attend at least 2 customer interviews
+#### **Collection** - Cymbal refactoring & Mobile support (Driver: <TeamMember name="Hugues Pouillot" />)
+* Expose frame resolution and issue processing as endpoints
```

Comment:
I feel like we're missing the plugin server work that will need to happen as part of this